aries_vcx/src/utils/mockdata/mockdata_connection.rs

```diff
@@ -2,7 +2,7 @@
 pub const ARIES_CONNECTION_INVITATION: &str = r#"
 {
     "@id": "28b39b79-f5db-4478-879a-15bb12632d00",
-    "@type": "did:sov:BzCbsNYhMrjHiqZDTUASHg;spec/connections/1.0/invitation",
+    "@type": "https://didcomm.org/connections/1.0/invitation",
```

Comment:
Do we test deserialization of the legacy format somewhere?

Reply:
yup I've created tests `test_deserialize_connection_ack_legacy, test_serialize_forward_message_to_legacy_format` to assure this PR is not introducing breaking change

Reply:
The test does not seem to test deserialization of the legacy format, however?

Reply:
fixed

Reply:
good catch! Must have find+replace by mistake. Fixed now